install/mysql/glpi-empty.sql

```diff
@@ -886,7 +895,6 @@ CREATE TABLE `glpi_items_disks` (
   KEY `mountpoint` (`mountpoint`),
   KEY `totalsize` (`totalsize`),
   KEY `freesize` (`freesize`),
-  KEY `itemtype` (`itemtype`),
   KEY `items_id` (`items_id`),
```

Comment:
There are multiple tables with ``KEY `item_id` (`items_id`)``. For some of them, it is related to an older schema, where relation was only on one item (e.g. `computers_id` has been replaced by `itemtype` + `items_id`, and key has been renamed from `computers_id` to `items_id`).

IMHO, these keys should be dropped, as no query using `items_id` without `itemtype` should be done.

Reply:
Indeed, just querying on items_id is a nonsense, those ones can be dropped

Reply:
I removed them.